examples/minimal/packages/client-react/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import { useComponentValue } from "@latticexyz/react";
+import { useMUD } from "./MUDContext";
+
+export const App = () => {
+  const {
+    components: { CounterTable },
+    singletonEntity,
+    network: { signer },
+    worldSend,
+  } = useMUD();
+
+  const counter = useComponentValue(CounterTable, singletonEntity);
+
+  return (
+    <>
+      <div>
+        Counter: <span>{counter?.value ?? "??"}</span>
+      </div>
+      <button
+        type="button"
+        onClick={async (event) => {
+          event.preventDefault();
+
+          // Create a World contract instance
+          const s = signer.get();
```

Comment:
this is not needed right?

Reply:
yep, I think you're right! `worldSend` handles this